src/components/Filters/index.js

```diff
@@ -0,0 +1,337 @@
+import { h, Component } from 'preact'
+import _ from 'lodash'
```

Comment:
Let's just import the methods we need, also I think currently best practice to `import clone from 'lodash/clone'` rather than relying on tree shaking with `{ clone }` https://advancedweb.hu/2017/02/07/treeshaking/